### PR TITLE
feat: match theme-color with primary-color

### PR DESCRIPTION
**Pull request by BastelKlug**
Closes: #406

Hi, I think the changes are self-explaining. 
Here is a demo video:

https://user-images.githubusercontent.com/44060099/179393694-a09d7429-03a4-4e2a-86b5-5c5831f70527.mp4

Signed-off-by: Kerim Bilgic <bastelklug@pfusch.eu>

### CODE REVIEW

**Review by pedrolamas — APPROVED** [review]
Thank you @BastelKlug, I tested on Windows 11 with Edge browser, working great! 🙂

**Review by matmen — APPROVED** [review]

**Comment by matmen** [comment]
neat, thanks!

File: vue.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = defineConfig({
     }
   },
   pwa: {
-    themeColor: '#2196F3',
+    themeColor: '',
```
Comment (pedrolamas) [inline vue.config.js:22]: @BastelKlug was there any specific reason for this line to change?

I ask this as I've noticed this warning on the console caused by it:

image

Comment (BastelKlug) [reply vue.config.js:22]: Yes, when I removed the line, it took the default value from the plugin. So I left it blank. Maybe there is another way 🤔

Comment (pedrolamas) [reply vue.config.js:22]: MS Edge in Win11 seems to work fine with the value in... What browser/phone did you notice that you needed to clear that? Might be the case here...

Comment (BastelKlug) [reply vue.config.js:22]: Chrome on Windows 11. Maybe it was a bug with the Vue Cli Hot reloading?

Comment (BastelKlug) [reply vue.config.js:22]: Chrome on MacOS, also does not work:

https://user-images.githubusercontent.com/44060099/181530634-372550b6-c623-44e8-ae4b-c375b877c059.mov

And also the bar in Safari that you see when you scroll up too high:
Bildschirmfoto 2022-07-28 um 16 17 19

Comment (pedrolamas) [reply vue.config.js:22]: Ah, I just tested with Chrome on Win11 and worked fine... guess this is a Mac thing only then?

**Pull request merged**